src/epcpm/importexportdialog.ui

```diff
@@ -181,6 +181,26 @@
     </widget>
    </item>
    <item row="8" column="0">
+    <widget class="QLabel" name="static_modbus_spreadsheet_label">
+     <property name="text">
+      <string>Static Modbus Spreadsheet:</string>
+     </property>
+     <property name="alignment">
+      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+     </property>
+    </widget>
+   </item>
+   <item row="8" column="1">
+    <widget class="QLineEdit" name="staticmodbus_spreadsheet"/>
+   </item>
+   <item row="8" column="2">
```

Comment:
Should all of the row's be identified with a descriptive const for their string numeric representation?
https://stackoverflow.com/questions/17440185/define-constants-in-xml

Reply:
Maybe? The `importexportdialog.ui` and all other `.ui` files are generated from the Qt Designer utility, so I don't have control over the format of the XML.

Reply:
Ahh ok gotcha.